packages/peregrine/lib/talons/AccountInformationPage/useAccountInformationPage.js

```diff
@@ -0,0 +1,122 @@
+import { useCallback, useMemo, useState, useEffect } from 'react';
+import { useMutation, useQuery } from '@apollo/client';
+import { useUserContext } from '../../context/user';
+
+export const useAccountInformationPage = props => {
+    const {
+        mutations: {
+            setCustomerInformationMutation,
+            changeCustomerPasswordMutation
+        },
+        queries: { getCustomerInformationQuery }
+    } = props;
+
+    const [{ isSignedIn }] = useUserContext();
+
+    const [activeChangePassword, setActiveChangePassword] = useState(false);
+    const [updatedData, setUpdatedData] = useState(false);
+    const [isUpdateMode, setIsUpdateMode] = useState(false);
+
+    const { data: accountInformationData, error: loadDataError } = useQuery(
+        getCustomerInformationQuery,
+        { skip: !isSignedIn, fetchPolicy: 'cache-and-network' }
```

Comment:
```suggestion
        {
            skip: !isSignedIn,
            fetchPolicy: 'cache-and-network',
            nextFetchPolicy: 'cache-first'
        }
```

In the v3 upgrade we had to start adding `nextFetchPolicy` to retain the original behavior of `cache-and-network`. Without it the update mutation triggers a refetch of this query even though we are requesting the modified data in the response of the mutation.